.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,57 @@
+{
+	"name": "Go",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update the VARIANT arg to pick a version of Go: 1, 1.16, 1.15
+			"VARIANT": "1.16",
+			// Options
+			"INSTALL_NODE": "true",
+			"NODE_VERSION": "lts/*"
+		}
+	},
+	"mounts": [
+		"source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind",
```

Comment:
Why does `docker.sock` need to be mounted?

Reply:
This is mounted so that you can run docker commands inside the container that call out to the host docker instance